src/themes/jekyll/_layouts/example.html

```diff
@@ -0,0 +1,25 @@
+---
+layout: page
+---
+
+<p>
+  {{ content }}
+</p>
+
+<p>
+  <h2>Explore the Code</h2>
+  {% include components/sw-include.html %}
+  {% include components/js-include.html %}
+  See all the details by opening your browser's
+  <a href="https://developers.google.com/web/tools/chrome-devtools/progressive-web-apps" target="_blank">Developer Tools</a>.
+</p>
+
+{% if page.link_to_workbox_sw %}
+<p>
+  <h2>See a Full Example</h2>
+  <p>
+    This is only part of the picture! <a href="">Learn more</a> about how module
```

Comment:
It will link to a not-yet-written `workbox-sw` example when it's available.

We're going to be shipping examples that show how each of the standalone modules can be used, but we're also going to have at least one main example tying everything together as part of `workbox-sw`. If someone comes to this page they wouldn't otherwise realize that there's a higher-level library that stitches the routing (or runtime caching, or cache expiration, etc.) together, and I want to give them that context.

Very happy to rephrase if you have any suggestions, though.